ocs_ci/framework/exceptions.py

```diff
@@ -3,3 +3,15 @@ class ClusterPathNotProvidedError(Exception):
 
     def __str__(self):
         return "Please provide a --cluster-path that is empty or non-existant."
+
+
+class ClusterNameLengthError(Exception):
+
+    def __init__(self, name, min, max):
+        self.name = name
+        self.min = min
+        self.max = max
+
+    def __str__(self):
+        return (f"Cluster Name '{self.name}' is {len(self.name)} characters long "
```

Comment:
```python
return (
    f"......"
    f"...."
)
```

Reply:
Just to clarify, should I fix the formatting here?

Reply:
just brackets https://ocs-ci.readthedocs.io/en/latest/docs/coding_guidelines.html

You have return (f"fsadfsa"

But we rather see the 
```
return (
    f"content on other line if it's too long above like you have"
) # bracket at the new line
```

Reply:
Make sense. Fixed with the last commit.